deploy/crds/operator.ibm.com_authentications_crd.yaml

```diff
@@ -274,6 +285,7 @@ spec:
               - imageName
               - imageRegistry
               - imageTag
+              - scopeClaim
```

Comment:
@rashmi43 does `scopeClaim` need to be under the `required` field? If so, we need to add it in the CR for `identity-provider`.

Reply:
I'm not sure of the details of the changes but I think it should be removed from here because it's part of the config